### PR TITLE
maint ~ configuration and utility clean-up and maintenance

### PR DESCRIPTION
**Pull request by rivy**
* waiting for the CI to pass to make sure I didn't make any silly mistakes.

### CODE REVIEW

File: .github/workflows/GnuTests.yml [hunk 1]
```diff
@@ -77,14 +77,14 @@ jobs:
       run: |
         ## Build binaries
         cd '${{ steps.vars.outputs.path_UUTILS }}'
-        bash util/build-gnu.sh
+        bash util/build-gnu.bash
```
Comment (sylvestre) [inline .github/workflows/GnuTests.yml:80]: why this change?
why not making them .sh compatible instead?

Comment (rivy) [reply .github/workflows/GnuTests.yml:80]: Yeah, I just jumped to the rename for `shellcheck`.

But I think it's just a variable expansion, so it should be reasonably simple to change. 
I'll look at it from that angle and try to rework it as POSIX-shell compatible.

Comment (rivy) [reply .github/workflows/GnuTests.yml]: Yep, dropping this rename (as unneeded) in the revision.
I must have been using an old `shellcheck` on a different PC; checking now shows no complaints.

**Comment by sylvestre** [comment]
I guess it is normal that it is still a draft :)

**Comment by rivy** [comment]
@sylvestre , well, that was a slog trying to iterate changes and debug a process which takes 1 to 2 hours for feedback. 😅 

I plan to change this to a PR in a couple of hours (when CI passes).

File: util/build-gnu.sh [hunk 2]
```diff
@@ -1,64 +1,86 @@
...
+
+cd "${path_UUTILS}" && echo "[ pwd:'${PWD}' ]"
+make PROFILE="${UU_MAKE_PROFILE}"
+cp "${UU_BUILD_DIR}/install" "${UU_BUILD_DIR}/ginstall" # The GNU tests rename this script before running, to avoid confusion with the make target
 # Create *sum binaries
-for sum in b2sum b3sum md5sum sha1sum sha224sum sha256sum sha384sum sha512sum
-do
-    sum_path="${BUILDDIR}/${sum}"
-    test -f "${sum_path}" || cp "${BUILDDIR}/hashsum" "${sum_path}"
+for sum in b2sum md5sum sha1sum sha224sum sha256sum sha384sum sha512sum; do
```
Comment (sylvestre) [inline util/build-gnu.sh:46]:
```suggestion
for sum in b2sum b3sum md5sum sha1sum sha224sum sha256sum sha384sum sha512sum; do
```

Comment (rivy) [reply util/build-gnu.sh:46]: 👍🏻

**Comment by rivy** [comment]
Added a commit fixing a `cargo clippy` complaint for `head` which slipped into the 'main' tree with a recent commit.
I'll refrain from further rebasing onto 'main'.

Any other comments/concerns?

**Comment by sylvestre** [comment]
sounds great :)

**Comment by rivy** [comment]
> and it was fixed by the initial author: #3124

I'll drop the fix commit and rebase off the new 'main'  then...

**Comment by rivy** [comment]
Done... rebased and pushed onto the fixed 'main'.
That should pass (🤞🏻) and then be merge-able if you concur.

**Comment by sylvestre** [comment]
excellent, thanks

**Comment by sylvestre** [comment]
More conflicts on https://github.com/uutils/coreutils/pull/3045 ;)

**Comment by rivy** [comment]
I'm on it.
